packages/components/src/styles/container/container.stories.ts

```diff
@@ -7,11 +7,12 @@ const { overrideArgs } = storybookHelpers('sd-container');
 const { generateTemplate } = storybookTemplate('sd-container');
 
 /**
- * Container lets users delimit and highlight a piece of content. The user has no interaction with it, it is merely a visual element that influences the flow of the page.
+ * Allows users delimit and highlight a piece of content. The user has no interaction with it, it is merely a visual element that influences the flow of the page.
```

Comment:
Allows users to delimit and highlight a piece of content that influences the flow of the page. The user has no interaction with it.

Reply:
Let’s shorten it to:

Used to enclose and highlight specific parts of content on a page.